Package.swift

```diff
@@ -1,10 +1,13 @@
-// swift-tools-version:4.0
+// swift-tools-version:5.5
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "ShowcaseKit",
+    platforms: [
+        .iOS(.v13)
```

Comment:
I think we could allow all the way down do iOS 11.
```suggestion
        .iOS(.v11)
```